administration/src/application/components/ApplyController.tsx

```diff
@@ -28,7 +31,7 @@ const SuccessContent = styled.div`
   margin-bottom: 1rem;
 `
 
```

Comment:
There's a variable `regionId` above, which should not longer be necessary. Would you mind deleting it?

Reply:
ok i think the query for dataPrivacyPolicy has to be moved to a position where we have the regionId